Source/WebCore/Modules/url-pattern/URLPattern.h

```diff
@@ -0,0 +1,95 @@
+// JBG TODO notes
```

Comment:
Please remove all these comments.

Reply:
Absolutely! This is just a reference branch / draft for myself + anyone who wants to see my WiP! Definitely don't intend to put these comments in main!